transform-plugins/README.rst

```diff
@@ -85,16 +78,15 @@ CSV Formatter
   Batch and
   Realtime
 :Description:
-  Formats a structured record as a CSV Record. CSV Record formats supported are DELIMITED, EXCEL, MYSQL, RFC4180 and TDF. When the format is DELIMITED one can specify different delimiters that a CSV record should use for separting fields. 
+  Formats a Structured Record as a CSV Record. Supported CSV Record formats are DELIMITED, EXCEL, MYSQL, RFC4180, and TDF. When the format is DELIMITED, one can specify different delimiters that a CSV Record should use for separating fields.
 :Configuration:
-    **format:** Specifies the format of the CSV record to be generated,
-    **delimiter:** Specifies the delimiter to be used to generate a CSV Record. This option is available when format is specified as DELIMITED and 
-    **schema:** Specifies the output schema. Output Schema should have only field of type String. 
+  - **format:** Specifies the format of the CSV Record to be generated.
+  - **delimiter:** Specifies the delimiter to be used to generate a CSV Record. This option is available when the format is specified as DELIMITED.
+  - **schema:** Specifies the output schema. Output schema should only have fields of type String.
```

Comment:
I think an example of an input record getting transformed into an output record along with format, delimiter and schema used will make things clearer.